mlbaklava/resources/deploy/main.py

```diff
@@ -14,12 +14,13 @@
 from flask import Flask, request
 import gunicorn.app.base
 
+from mlsriracha.deploy import DeployAdapter
```

Comment:
I am assuming mlsriracha provides data adapter for containers build via baklava. 

Can we port over mlsriracha code to this repo? Less maintenance and easy to integrate

Reply:
Are there any other reasons to keep mlsriracha separate?